resources/views/designer/designer.blade.php

```diff
@@ -1,7 +1,7 @@
 @extends('layouts.layout')
 @section('content')
     <div id="designer-container">
-      <designer process-uid="{{$process->uid}}"></designer>
+      <designer process-uid="{{$process->uid}}" process-title="{{ $title }}"></designer>
```

Comment:
Can't we just use $process->title?

Reply:
actually process-> name and $title have the same value.

"$title" is the title in the tab browser, please  let me know if I should have another text for title